x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/delete.ts

```diff
@@ -302,28 +302,23 @@ export default function deleteConnectorTests({ getService }: FtrProviderContext)
     }
 
     it('should delete a connector with an unsupported type', async () => {
+      const { space, user } = SuperuserAtSpace1;
       await kibanaServer.importExport.load(
-        'x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/fixtures/unsupported_connector_type.json'
+        'x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/fixtures/unsupported_connector_type.json',
+        { space: space.id }
       );
 
-      const { space, user } = SuperuserAtSpace1;
-      const { body: createdConnector } = await supertest
-        .post(`${getUrlPrefix(space.id)}/api/actions/connector`)
-        .set('kbn-xsrf', 'foo')
-        .send({
-          name: 'My Connector',
-          connector_type_id: 'test.index-record',
-          config: {
-            unencrypted: `This value shouldn't get encrypted`,
-          },
-          secrets: {
-            encrypted: 'This value should be encrypted',
-          },
-        })
-        .expect(200);
+      const res = await supertestWithoutAuth
+        .get(`${getUrlPrefix(space.id)}/api/actions/connectors`)
+        .auth(user.username, user.password);
+
+      const invalidConnector = res.body.find(
+        (connector: { connector_type_id: string; id: string }) =>
+          connector.connector_type_id === '.invalid-type'
+      );
 
       const response = await supertestWithoutAuth
-        .delete(`${getUrlPrefix(space.id)}/api/actions/connector/${createdConnector.id}`)
+        .delete(`${getUrlPrefix(space.id)}/api/actions/connector/${invalidConnector.id}`)
```

Comment:
nit: you can hardcode the ID if you want.

Reply:
That is what I tried first but the ID from the import was not working :| I thought I was going crazy debugging it.

Reply:
Oh, interesting. Probably, the import changes the ID. All good.

Reply:
Yeah, thats what I thought so then I tried removing the ID from the fixtures file and the import operation fails without it which is even weirder.